cmd/buildkitd/main_unix.go

```diff
@@ -47,8 +48,14 @@ func listenFD(addr string, tlsConfig *tls.Config) (net.Listener, error) {
 	return nil, errors.New("not supported yet")
 }
 
-func traceSocketPath(root string) string {
-	return filepath.Join(root, "otel-grpc.sock")
+func traceSocketPath(inUserNS bool) string {
```

Comment:
This should be moved to https://github.com/moby/buildkit/blob/master/util/appdefaults/appdefaults_unix.go